docs/topics/ref_new-features-7-2-0.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.1.0/master.adoc
+
+:_template-generated: 2024-12-04
+:_mod-docs-content-type: REFERENCE
+
+[id="new-features-7-2-0_{context}"]
+= New features
+
+This section provides the new features and improvements of the {ProductFullName} 7.2.0.
+
+.Containerless CLI
+
+You can perform an application analysis for Java applications by using the {ProductShortName} CLI that does not require installation of a container runtime.
```

Comment:
A suggestion: Direct to the CLI guide to see more details on this feature.

Reply:
Will do when we have published

Reply:
When linking the CLI, please specify that the user has some installation prereqs to satisfy. I.e..

"See <link> to learn more about installation requirements" or something like that

Reply:
Will do this after publishing.